RecoEcal/EgammaClusterAlgos/src/SCEnergyCorrectorSemiParm.cc

```diff
@@ -51,6 +51,7 @@ SCEnergyCorrectorSemiParm::SCEnergyCorrectorSemiParm()
       caloGeom_(nullptr),
       isHLT_(false),
       isPhaseII_(false),
+      regTrainedWithPS_(true),
```

Comment:
@ram1123 please change this to `false`. As far as I understood this should restore things to the way they are now.
Once we have the new conditions validate we (@cms-sw/alca-l2) we'll take care of changing it to `true` when we update the GT.